solr/solrj/src/test/org/apache/solr/client/solrj/io/stream/expr/InjectionDefenseTest.java

```diff
@@ -17,12 +17,10 @@
 
 package org.apache.solr.client.solrj.io.stream.expr;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-
+import org.apache.solr.SolrTestCase;
 import org.junit.Test;
 
-public class InjectionDefenseTest {
+public class InjectionDefenseTest extends SolrTestCase {
```

Comment:
Why?

Reply:
Get all the benefits of the underlying randomization and thread leak checks from Lucene. This ensures that all these tests have a common base. There are no real downsides to doing this either.